client/Search.jsx

```diff
@@ -9,8 +11,9 @@ class Search extends React.Component {
   render() {
```

Comment:
If your component is not utilizing state, consider changing the component to be a stateless component instead of a stateful component.

Reply:
Refactor goal!